ruby/ext/google/protobuf_c/message.c

```diff
@@ -1263,13 +1263,17 @@ VALUE build_module_from_enumdesc(VALUE _enumdesc) {
   int n = upb_EnumDef_ValueCount(e);
   for (int i = 0; i < n; i++) {
     const upb_EnumValueDef* ev = upb_EnumDef_Value(e, i);
-    const char* name = upb_EnumValueDef_Name(ev);
+    char* name = strdup(upb_EnumValueDef_Name(ev));
```

Comment:
This `strdup()` call needs a corresponding call to `free()` after it's used in the call to `rb_define_const()`.

Reply:
Updated at 1314fb9747a2da9b6c61f9bd1d3c22aeb05142a7.